### PR TITLE
check generated file as part of GitHub Workflow

### PR DESCRIPTION
**Pull request by honnix**
## Description

This PR configures a step of GitHub Workflow to ensure that in case of modification of kustomize code, `make kustomize` was executed and the newly generated `flyte_generated.yaml` was added as a part of the commit.

This PR is inspired by https://github.com/lyft/flyteidl/blob/master/generate_protos.sh#L58

## Motivation

It is very likely that a new contributor of this repo updated some kustomize file but didn't do `make kustomize`. Having an automated step telling what is missing would be useful.

## How did I test it

* https://github.com/lyft/flyte/pull/200/checks?check_run_id=496295498 as a successful workflow execution
* https://github.com/lyft/flyte/pull/200/checks?check_run_id=496307946 as a failed one

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -15,6 +15,8 @@ jobs:
         go-version: [1.10]
     steps:
     - uses: actions/checkout@v1
+    - name: Kustomize and diff
```
Comment (wild-endeavor) [inline .github/workflows/tests.yml]: cool.  it seems this runs as part of this PR too.

Comment (honnix) [reply .github/workflows/tests.yml]: Yeah that was a successful execution.

**Comment by EngHabu** [comment]
Thank you!

**Comment by honnix** [comment]
_Sorry I didn't know force push would remove previous workflow executions._ :(

**Pull request merged**